src/mesh.jl

```diff
@@ -0,0 +1,43 @@
+function load_mesh()
+return quote
+
+using .Polyhedra: Mesh
```

Comment:
If `Polyhedra` is not loaded, this would crash. This requires a two-level check in both the `Polyhedra` and `Makie` loading. Whichever is loaded later should execute this `load_mesh` function. See [here](https://github.com/JuliaReach/Reachability.jl/issues/449#issuecomment-462451906) for a proposal.